src/HH4b/postprocessing/CreateDatacard.py

```diff
@@ -332,6 +332,20 @@
         pass_only=True,
         convert_shape_to_lnN=True,
     ),
+    "FSRPartonShower": Syst(name="ps_fsr", prior="shape", samples=sig_keys),
+    "ISRPartonShower": Syst(name="ps_isr", prior="shape", samples=sig_keys),
+    "scale": Syst(
+        name=f"{CMS_PARAMS_LABEL}_QCDScaleacc",
+        prior="shape",
+        samples=sig_keys + ["ttbar"],
```

Comment:
@cmantill right now it looks like we're not saving the scale weights for ttbar. Let's make sure they're saved for ttbar in the next production